### ISSUE TITLE
Configuration - Service Labels

### ISSUE THREAD

**Issue by danielmackay** [event-catalog/eventcatalog#546]
## Context / Problem

Currently, Event Catalog is geared around Microservices.  'Services' being a key part of the system.

There is a large overlap with a Microservice system and a Modular Monolith.  In this instance a 'Service' is very similar to a 'Module' in a Modular Monolith

## Suggestion

I read in the documentation that 'Services' are a generic term, but it would be great to have some control over this.

It would be great if I could override this label to be called 'Modules' instead, which would save a lot of confusion when documenting a Modular Monolith application using Event Catalog.

**Comment by black-snow** [comment]
Do you have a modulith and other services at the same time? Do you need both?

**Comment by danielmackay** [comment]
> Do you have a modulith and other services at the same time? Do you need both?

Yes, it is possible that you could have both.  You might have a Modular Monolith with modules mostly communication together, but some modules could communicate with microservices 

Perhaps 'Services' could be extended with a 'Type' field to allow to to be either a Module or a Service ?

**Comment by boyney123** [comment]
I think https://github.com/event-catalog/eventcatalog/issues/2539 will help here, going to close this issuein favour for that. But introducing a system might be able to help you group things together.
